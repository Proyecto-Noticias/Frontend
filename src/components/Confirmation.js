///// START CODE REVIEW COMMENT

// Amazing!

///// END CODE REVIEW COMMENT

import React from "react";
import { Link } from "react-router-dom";

const Confirmation = () => {
  return (
    <div className="confirmation__container">
      <div className="confirmation__deco"></div>
      <div className="confirmation__text">
        <p>Your account has been activated</p>

        <Link className="confirmation__link" to="login">
          Go to login
        </Link>
      </div>
    </div>
  );
};

export default Confirmation;
