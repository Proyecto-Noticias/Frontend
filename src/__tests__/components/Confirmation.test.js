import React from 'react';
import { shallow } from "enzyme";
import  Confirmation  from "../../components/Confirmation";



describe('Test of <Confirmation />', () => {
  const wrapper = shallow( <Confirmation /> )
  
  test('should show the component correctly', () => {   
    expect ( wrapper ).toMatchSnapshot()
  })
  
  test('should have a title', () => {
    
    const p = wrapper.find('p')
    expect( p.text()).toBe('Your account has been activated') 

  })

  test('should have a link to home', () => {
    
    const Link = wrapper.find('Link')
    expect( Link.text()).toBe('Go to login') 

  })
  

})
