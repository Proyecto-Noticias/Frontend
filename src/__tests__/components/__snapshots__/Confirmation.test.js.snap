// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <Confirmation /> should show the component correctly 1`] = `
<div
  className="confirmation__container"
>
  <div
    className="confirmation__deco"
  />
  <div
    className="confirmation__text"
  >
    <p>
      Your account has been activated
    </p>
    <Link
      className="confirmation__link"
      to="login"
    >
      Go to login
    </Link>
  </div>
</div>
`;
